test/built-ins/Proxy/apply/call-result.js

```diff
@@ -0,0 +1,23 @@
+// Copyright (C) 2015 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 9.5.13
+description: >
+    Return the result from the trap method.
+info: >
+    [[Call]] (thisArgument, argumentsList)
+
+    9. Return Call(trap, handler, «target, thisArgument, argArray»).
+---*/
+
+var target = function(a, b) { return a + b; };
+var handler = {
+    apply: function(t, c, args) {
+        return 2 * t.apply(null, args);
```

Comment:
This test duplicates coverage with `call-parameters.js`. You can simplify it by declaring `var result = {};` at the top of the test, returning `result` from the trap, and asserting `sameValue(p.call(), result)`